src/popup/sass/layout/_tooltip.scss

```diff
@@ -0,0 +1,61 @@
+.tooltip {
+  position: relative;
+  display: inline-block;
+}
+
+.tooltiptext {
+  visibility: hidden;
+  position: absolute;
+  width: 180px;
+  background-color: #fff;
+  border: 2px solid #d8d8d8;
+  z-index: 400;
+  width: 20rem;
+  box-shadow: 10px 10px 10px -10px rgba(0, 0, 0, 0.25);
+  color: #333;
+  text-align: left;
+  padding: 5px;
+  opacity: 0;
+  transition: opacity 0.3s;
+}
+
+.question-color {
+  color: #d8d8d8 !important;
+}
```

Comment:
Please don't use `!important` here. You can use [bulma color helpers](https://bulma.io/documentation/helpers/color-helpers/) in the `HTML` itself..